### ISSUE TITLE
".infinite-scroll-last" not appearing on last page load

### ISSUE THREAD

**Issue by danof** [metafizzy/infinite-scroll#706]
The "last" event appears to be firing fine. However, ".infinite-scroll-last" doesn't get set to "display:block". Doesn't appear to be working in your demos.

**Comment by desandro** [comment]
Thank you for reporting this bug! This will be fixed in the next release.

**Comment by satrya** [comment]
I have the same issue with one of my client site, please take a look http://ilovechrisbaker.com/. If you click the Load More Projects to the end.
1. The button still appear
2. No message the end of content.

_*nb: I have permission to show the website_

**Comment by desandro** [comment]
v3.0.2 has been released with this fix added. Please update! 🌈 🐻

**Comment by sharifzadesina** [comment]
@desandro This is not fixed dude.

**Comment by sharifzadesina** [comment]
@desandro You can also check this in your own demos.

**Comment by icetee** [comment]
Same here

**Comment by desandro** [comment]
@icetee @mohammadjalili Can you please provide **reduced test case** or a **live URL** so I can verify this is an issue?  See [Submitting Issues in the contributing guidelines](https://github.com/metafizzy/infinite-scroll/blob/master/.github/contributing.md#submitting-issues).

**Comment by sharifzadesina** [comment]
@desandro Why you can just take a look at your own demos?

**Comment by desandro** [comment]
I'm not able to reproduce this issue. On https://infinite-scroll.com/demo/full-page/

![screen shot 2018-02-02 at 1 51 03 pm](https://user-images.githubusercontent.com/85566/35749616-2981b4fc-0820-11e8-8a9c-51691392b705.png)

**Comment by danigore** [comment]
Hy guys!
Primarly thank you very much your all work DeSandro!
Anyway here is a small bug, really when the user reaching the last page the .infinite-scroll-last get display:block, but the parent .page-load-status will be display:none immediately, so the child elem also disappearing... But this can be easy done with a forced block on the parent in the css... This time turning up an another problem on such pages, which never loading new content after the ready event (so on pages which dont have paginated content), because this time remains in the default state the all children (so their dont have any display value), because of this will be visible the all status elem.

You can solve this bug easy, just force your css like this, and the status will be working perfectly:
```
.page-load-status {
	display:block !important; /*never problem the forced block, because the all children will be display:none, exception that one, what handle the infinite-scroll (so the active status)*/
}
.infinite-scroll-request,
.infinite-scroll-error {
	display:none;
}
.infinite-scroll-last {
	display:block; /*Needful the the block value because the pages without pagination (but non forced!) */
}
```

**Comment by jruns** [comment]
I was having similar issues as described above. Even though I set the 'status' option it wasn't showing the last message when reaching the last page. And if I tried to unhide it manually on the last.infiniteScroll event triggering it was still getting hidden after I ran my code. So I solved it for myself by adding a class to the status container and then setting a little css to force the last message to be displayed. A little more elegant IMHO than some of the above suggestions.

```
$container.on( 'last.infiniteScroll', function( event, response ) {
	jQuery('#scroller-status').addClass( 'last-page' );
});
```

```
.scroller-status {
	display: none;
}
.scroller-status.last-page, .scroller-status.last-page .infinite-scroll-last {
	display: block !important;
}
```
